OpenTkControl/OpenTkControlBase.xaml.cs

```diff
@@ -305,6 +305,8 @@ public Task RequestRepaint()
             return tcs.Task;
         }
 
+        protected bool IsDesignMode => DesignerProperties.GetIsInDesignMode(this);
```

Comment:
I think inlining this would be more clear

Reply:
modified